content/50.discussion.md

```diff
@@ -16,5 +16,6 @@ Some concerns include:
 1. Who is being included in clinical trials?
 2. While biotechnological developments are exciting, how likely are they to be available to people broadly?
 3. Even this review is likely biased (e.g., we look at traditional western medicine in the nutraceuticals section and mainly report statistics from US and other data collection services)
- 
 
```

Comment:
@tlukan @shiktadas Since I think you guys may be working on rewriting this, please refer to #532 for what I think may be some important considerations/suggestions for this part of the Discussion.